components/calendar/src/any_calendar.rs

```diff
@@ -1045,7 +1254,7 @@ mod tests {
         let month = types::MonthCode(month_code.parse().expect("month code must parse"));
 
         let date = Date::try_new_from_codes(era, year, month, day, calendar).unwrap_or_else(|e| {
-            panic!(
+            self::panic!(
```

Comment:
Question: what does `self::panic!` do and why do you need it?

Reply:
Im not sure what it does but I just know I have to use self::panic because I was getting an ambiguity error.

Reply:
Please use `panic` here and un-import `core::panic`. You got that error because you (or perhaps your IDE) decided to import `core::panic`.

Reply:
done, thanks :)